docs/imexam/walkthrough.rst

```diff
@@ -81,7 +81,7 @@ displays the same image to each one::
 
 
 It's also possible to load a FITS image object that you already have opened in your
-python session, if no extension is given, then the first IMAGE exension that is found
+python session, if no extension is given, then the first IMAGE extension that is found
```

Comment:
I know there's an XTENSION thing nearby, but I think this use can use the normal spelling.